### PR TITLE
add applyconfig periodic jobs for all clusters

### PR DESCRIPTION
**Pull request by droslean**
/cc @openshift/openshift-team-developer-productivity-test-platform 

Signed-off-by: Nikolaos Moraitis <nmoraiti@redhat.com>

### CODE REVIEW

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -717,15 +717,15 @@ periodics:
           secretKeyRef:
             key: url
             name: ci-slack-api-url
-      image: registry.svc.ci.openshift.org/ci/applyconfig:latest
+      image: applyconfig:latest
```
Comment (petr-muller) [inline ci-operator/jobs/infra-periodics.yaml:720]: What makes `applyconfig` avaialble on app.ci? Even the presubmits and postsubmits are using full pullspec.

Comment (droslean) [reply ci-operator/jobs/infra-periodics.yaml:720]: The imagestream exists in the `ci` namespace and its `local: true` as well

Comment (petr-muller) [reply ci-operator/jobs/infra-periodics.yaml:720]: That's not what I asked... I'm trying to understand on what are we relying on here. The IS does not seem to have a manifest in the repo, what is making sure it exists?

Comment (alvaroaleman) [reply ci-operator/jobs/infra-periodics.yaml:720]: This is from the test-image distributor, which is configured to distribute everything in the `ci` namespace: https://github.com/openshift/release/tree/b2770007d21ea33f7ac1ea2ad780904e127448b9/clusters/app.ci/prow

**Review by stevekuznetsov — COMMENTED** [review]
For all of our other periodic jobs we prefix the name with `periodic-` -- can we follow that pattern here?

**Review by stevekuznetsov — APPROVED** [review]

**Pull request merged**